bin/src/trust-dns.rs

```diff
@@ -401,7 +401,7 @@ fn main() {
         info!("binding UDP to {:?}", udp_socket);
         let udp_socket = runtime
             .block_on(UdpSocket::bind(udp_socket))
-            .unwrap_or_else(|_| panic!("could not bind to udp: {}", udp_socket));
+            .unwrap_or_else(|err| panic!("could not bind to udp: {udp_socket}: {err}"));
```

Comment:
Sorry, would still like to get rid of the first colon.

Reply:
Oh, I didn't notice that.